packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,75 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
+    padding: 0;
+  }
+
+  .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Helvetica Neue');
```

Comment:
Why is font-family being set here? This be inherited from font-family being set on the [HTML element](https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.scss#L8)?

Reply:
Good point, updated to inherit.